exporter/kafkaexporter/jaeger_marshaler_debug.go

```diff
@@ -113,6 +126,62 @@ func (j jaegerMarshalerDebug) spanAsString(span *jaegerproto.Span) string {
 	return sb.String()
 }
 
+func (j jaegerMarshalerDebug) cureSpan(span *jaegerproto.Span, topic string) (*sarama.ProducerMessage, error) {
```

Comment:
TODO: I will need to change this into a loop in case the first time we truncate the attributes does not cause it to go down below 1MB. In each iteration, we can half the attribute value size. We can try a max of maybe 5 times before giving up.

Reply:
done